packages/account/src/Helpers/utils.ts

```diff
@@ -281,8 +281,23 @@ export const isAdditionalDocumentValid = (document_type: FormikValues, document_
 
 export const isDocumentNumberValid = (document_number: string, document_type: FormikValues) => {
     const is_document_number_invalid = document_number === document_type.example_format;
-    if (!document_number) {
-        return localize('Please enter your document number. ') + getExampleFormat(document_type.example_format);
+    if (!document_number && document_type.text) {
+        let document_name = '';
+        switch (document_type.text) {
+            case 'Drivers License':
+                document_name = 'Driver License';
+                break;
+            case 'Social Security and National Insurance Trust':
+                document_name = 'SSNIT';
+                break;
+            default:
+                document_name = 'document number';
+                break;
+        }
+        return (
+            localize('Please enter your {{document_name}}. ', { document_name }) +
```

Comment:
Do not use variables in `localize` it will break the translation. 
ref: docs/JavaScript/README.md